quickbudget/api/permissions.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import permissions
+
+from quickbudget.models import Budget
+
+
+class BudgetMembersOnly(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        if request.user.is_superuser:
+            return True
+
+        if request.user in obj.members.all():
```

Comment:
I'd add checking if a user is a budget owner/creator.
+
I'd rename the class. A superuser or a budget creator could not be a budget member. It's more comprehensive to call it: "CanAccessBudget".
+ 
currently, a budget creator can delete all members (including themself), so they cannot access the budget after that.